spec/commands/add_spec.rb

```diff
@@ -106,4 +106,46 @@
     bundle "add 'baz' --source='file://does/not/exist'"
     expect(out).to include("Could not fetch specs from file://does/not/exist/")
   end
+
+  context "throws an error if a gem is added which is already specified in Gemfile with version" do
+    it "with different version requirement" do
```

Comment:
Your test description sounds more like a context for the tested behaviour rather than what the behaviour should be. This also applies to the other ‘it’ blocks in this PR